lib/rules/max-nesting-depth/__tests__/index.js

```diff
@@ -110,6 +110,38 @@ testRule(rule, {
   ]
 });
 
+testRule(rule, {
+  ruleName,
+  config: [1, { ignore: ["blockless-pseudoclasses"] }],
+
+  accept: [
+    {
+      code: "a { b { top: 0; }}"
+    },
+    {
+      code: "a { b { &:hover { top: 0; }}}"
+    },
+    {
+      code: "a { b { &:nest { &:nest-lvl2 { top: 0; }}}}"
```

Comment:
I believe this should be rejected as there are two levels here.

Reply:
```
a {
  b { // nesting level 1
    &:nest { // pseudo-class (parent depth level 1 taken)
      &:nest-lvl2 { // pseudo-class (parent depth level 1 taken)
        top: 0;
      }
    }
  }
}
```